### DISCUSSION TITLE
请问有tim或者QQ的appage版本吗

### DISCUSSION THREAD

**Discussion by chioview** [Gamuxorg/bbs#201] [投递和请求游戏]
游戏名称：
游戏名称(英文)：
下载地址 :（如果有的话）：
你好，github上的qq，下载了运行不了，请问这个可以做吗

**Comment by AD8099** [comment]
感觉deepin的qq还行

**Comment by wangling12** [comment]
ubuntu 的话一般用deepinqq https://github.com/zq1997/deepin-wine
appimage的版本作者应该是没时间维护了https://github.com/askme765cs/Wine-QQ-TIM

**Comment by wuyns** [comment]
qq如果只是聊天的话可以用icalingua，github里面就有

**Comment by Zhaoym233** [comment]
icalingua是个停更的第三方客户端（不过够用）  
gtk-talk是一个未完工的第三方客户端  
linuxqq是08奥运纪念版  
deepin wine qq/deepin wine tim是运行在deepin wine兼容层上的

**Comment by liulitchi** [comment]
icalingua 挂了， 有一个 icalingua++ 的项目。
